pages/linux/vkpurge.md

```diff
@@ -0,0 +1,17 @@
+# vkpurge
+
+> List or remove old kernel versions left behind by `xbps`.
+> The `version` arguments support shell globs.
+> More information: <https://man.voidlinux.org/vkpurge.8>.
```

Comment:
This command is specific to the Void Linux distribution, so I think we have to mention it.
See the related command `xbps` command description: 
https://github.com/tldr-pages/tldr/blob/90145116317305e1a477111e55afc473d5bee6d7/pages/linux/xbps.md?plain=1#L3C1-L3C89

Reply:
`xbps` is Void-specific, and the page mentions that the command exists on an `xbps`-managed OS.
In case that's not clear enough, we could add something like that:
> List or remove old kernel versions left behind by `xbps` ***(the Void Linux package manager)***.

but I don't think that's needed, as you can just run `tldr xbps` and see a better description.